taxi/forms.py

```diff
@@ -10,11 +10,12 @@ class CarForm(forms.ModelForm):
     drivers = forms.ModelMultipleChoiceField(
         queryset=get_user_model().objects.all(),
```

Comment:
The queryset for the `drivers` field in `CarForm` currently includes all users. If your user model includes non-driver users, this might not be appropriate. Consider filtering the queryset to include only users who are drivers. You might need to use a specific field or method to filter only driver users.